css/styles.css

```diff
@@ -0,0 +1,110 @@
+body  {
+  font-family: Georgia, Helvetica, serif;
+  text-align: center;
+  background-color: #FDF5E6;
+}
+
+h1  {
+  border-bottom: solid 5px #811818;
+  margin: 15px 5px 15px 5px;
+  padding: 15px;
+  font-size: 40px;
+}
+
+.sub-bio  {
+  font-family: "Palatino Linotype";
+}
+
+.used-link  {
+  font-style: italic;
+  text-decoration: none;
+}
+
+hr  {
+  margin: 15px;
+}
+
+img {
+  border: groove #A91717 10px;
+  width: 650px;
+  height: 400px;
+  margin: 10px;
+}
+
+.voldemort  {
+  text-decoration: line-through;
+  font-weight: bolder;
+  font-style: italic;
+}
+
+#bio-pics1 {
+  border: none;
+  margin: 20px 15px 15px 15px;
+  width: 250px;
+  height: 250px;
+}
+
+#bio-pics2  {
+  border: none;
+  margin: 20px 15px 15px 15px;
+  width: 350px;
+  height: 250px;
+}
+
+#bio-pics3 {
+  border: none;
+  margin: 20px 15px 15px 15px;
+  width: 250px;
+  height: 250px;
+}
+
```

Comment:
I condensed the original "bio-pics1" and "bio-pics3" into one class called "bio-pics", but left "bio-pics2" alone since it has a different width than the other two pictures.